search/tests/tests.py

```diff
@@ -53,6 +53,11 @@ def remove(self, doc_type, doc_id, **kwargs):
 @override_settings(SEARCH_ENGINE="search.tests.mock_search_engine.MockSearchEngine")
 @override_settings(ELASTIC_FIELD_MAPPINGS={"start_date": {"type": "date"}})
 @override_settings(MOCK_SEARCH_BACKING_FILE=None)
+@override_settings(EVENT_TRACKING_PROCESSORS=[
+    {
+        'ENGINE': 'eventtracking.django.tests.test_configuration.NopProcessor'
+    }
+])
```

Comment:
This setting override shouldn't be necessary, are you seeing an error if you don't include it?